bigframes/core/compile/scalar_op_compiler.py

```diff
@@ -1074,9 +1073,10 @@ def geo_st_intersection_op_impl(x: ibis_types.Value, y: ibis_types.Value):
     )
 
 
-@scalar_op_compiler.register_unary_op(ops.geo_x_op)
-def geo_x_op_impl(x: ibis_types.Value):
-    return typing.cast(ibis_types.GeoSpatialValue, x).x()
+@scalar_op_compiler.register_unary_op(ops.GeoStLengthOp, pass_op=True)
```

Comment:
Bad merge maybe? This shouldn't have been removed.

```suggestion
@scalar_op_compiler.register_unary_op(ops.geo_x_op)
def geo_x_op_impl(x: ibis_types.Value):
    return typing.cast(ibis_types.GeoSpatialValue, x).x()


@scalar_op_compiler.register_unary_op(ops.GeoStLengthOp, pass_op=True)
```